docs/user-guide/kubectl-overview.md

```diff
@@ -33,160 +33,271 @@ Documentation for other releases can be found at
 		$ kubectl get replicationcontroller <rc-name>
 
-		// List all replication controllers and services together in ps output format.
+		// List all replication controllers and services together in plain-text output format.
 		$ kubectl get rc,services
 
-3. `kubectl describe` - Show details of a specific resource or group of resources
+ * `kubectl describe` - Display all the details for one or more resources.
 
-		// Describe a node
+		// Display the details of the node with name <node-name>.
 		$ kubectl describe nodes <node-name>
 
-		// Describe a pod
+		// Display the details of the pod with name <pod-name>.
 		$ kubectl describe pods/<pod-name>
 
-		// Describe all pods managed by the replication controller <rc-name>
-		// (rc-created pods get the name of the rc as a prefix in the pod the name).
+		// Display the details of all the pods that are managed by the replication controller named <rc-name>.
+		// Remember: Any pods that are created by the replication controller get prefixed with the name of the replication controller.
 		$ kubectl describe pods <rc-name>
 
-4. `kubectl delete` - Delete resources by filenames, stdin, resources and names, or by resources and label selector
+ * `kubectl delete` - Delete resources either by filenames, label selectors, names, resource selectors, resources, or stdin.
 
-		// Delete a pod using the type and name specified in pod.yaml.
+		// Delete a pod using the type and name specified in the pod.yaml file.
 		$ kubectl delete -f pod.yaml
 
-		// Delete pods and services with label name=<label-name>.
+		// Delete all the pods and services that have the label name=<label-name>.
 		$ kubectl delete pods,services -l name=<label-name>
 
-		// Delete all pods
+		// Delete all pods.
 		$ kubectl delete pods --all
 
-5. `kubectl exec` - Execute a command in a container
+ * `kubectl exec` - Execute a command in a container.
 
-		// Get output from running 'date' from pod <pod-name>, using the first container by default.
+		// Get output from running 'date' from pod <pod-name>. By default, output is from the first container.
 		$ kubectl exec <pod-name> date
 
-		// Get output from running 'date' in <container-name> from pod <pod-name>.
+		// Get output from running 'date' in container <container-name> of pod <pod-name>.
 		$ kubectl exec <pod-name> -c <container-name> date
 
-		// Get an interactive tty and run /bin/bash from pod <pod-name>, using the first container by default.
+		// Get an interactive TTY and run /bin/bash from pod <pod-name>. By default, output is from the first container.
 		$ kubectl exec -ti <pod-name> /bin/bash
 
-6. `kubectl logs` - Print the logs for a container in a pod.
+ * `kubectl logs` - Print the logs for a container.
 
-		// Returns snapshot of logs from pod <pod-name>.
+		// Return the snapshot of the logs from pod <pod-name>.
```

Comment:
"a snapshot of logs", not "the snapshot". Every time you run the command it's equivalent to snapshotting current log output.